test/unit/Command/DownloadCommandTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Php\PieUnitTest\Command;
+
+use Php\Pie\Command\DownloadCommand;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[CoversClass(DownloadCommand::class)]
+class DownloadCommandTest extends TestCase
+{
+    public function testDownloadCommand(): void
+    {
+        $input  = $this->createMock(InputInterface::class);
+        $output = $this->createMock(OutputInterface::class);
```

Comment:
I wonder why you're using a mock here? Wouldn't be passing `StringInput('')` and `NullOutput` be sufficient?